src/test/java/seedu/uninurse/logic/commands/AddTaskCommandTest.java

```diff
@@ -130,10 +130,10 @@ public void equals() {
         // null -> returns false
         assertNotEquals(null, addTaskFirstCommand);
 
-        // different person index -> returns false
+        // different task -> returns false
         assertNotEquals(addTaskFirstCommand, addTaskSecondCommand);
 
-        // different task index -> returns false
+        // different patient index -> returns false
```

Comment:
I think "task" and "patient index" should be swapped. Should probably change the names of the `AddTaskCommand`s to be more fitting like the one in `AddConditionCommandTest`.

Reply:
Thanks for pointing that out. I've updated the PR with your suggestions.